stroeer/page/section/v1/section_page.proto

```diff
@@ -17,12 +17,20 @@ message SectionPage {
 
 // All meta information to render a section page
 message Section {
-  // References to get access to the section on different technical platforms like
-  // canonical web or AMP (required).
-  stroeer.core.v1.References references = 1;
+  // Generic map containing general content and configuration information of
+  // the section (required).
+  //
+  // This map will always contain non empty values for the following keys:
+  //
+  // * `url_path`: URL path for this section e.g. /sport/fussball/bundesliga/
+  // * `url_canonical`: Canonical URL of the section, may differ if external, e.g. https://www.giga.de/tech/
+  // * `url_amp`: AMP URL of the section
```

Comment:
We could talk about this mandatory field names

Reply:
Maybe I need to see an example, but my first thought was that those URLs are represented by the `Category` with it's reference?!

Reply:
I removed the message `References`:

```proto
// References to get access to the article on different technical platforms like
// canonical web or AMP.
message References {
  // URL path for this article e.g. /section/id_$ID/title.html (required).
  string path = 1;

  // Canonical URL of the article, may differ if external, e.g. https://www.giga.de/external.html
  // (required).
  string canonical = 2;

  // AMP URL of the article.
  string amp = 3;
}
```

These URLs are now part of the main `fields` and marked as mandatory like the `headline`. Because of:

- removing non-reusable messages
- "mandatory data shouldn't be in fields" is not an case anymore